middleware_timeout.go

```diff
@@ -22,21 +22,14 @@ func Timeout(timeout time.Duration) contractshttp.Middleware {
 		done := make(chan struct{})
 
 		go func() {
-			defer func() {
-				if r := recover(); r != nil {
-					LogFacade.Request(ctx.Request()).Error(r)
```

Comment:
The log is required, please keep it. I saw it was removed in gin as well, please add it back.